[COMSUP-219] Node 20 support (#960)
edgio-docs/edgio-docs: src/guides/v7/performance/cdn_as_code/edgio_config.md

The `cloudRuntime` key (string) determines which version of Node.js will run your app on our platform. Supported values are:

- `nodejs16.x` (v7.4.0 through v7.4.4)
- `nodejs18.x` (v7.4.0 or later)
- `nodejs20.x` (v7.5.0 or later)

> If the `cloudRuntime` key is not defined, then {{ PRODUCT }} will detect your project's Node.js version upon running `{{ CLI_CMD(deploy) }}`. If an unsupported version is detected, then it will set your version to `nodejs18.x`. For instance:

Author: Confirmed that with Node.js 20 support, the default is still 18.x:

```
const validRuntimesSet = new Set(['nodejs18.x', 'nodejs20.x'])
// Default cloud runtime in case when invalid one is detected locally / set through cloudRuntime property.
export const DEFAULT_CLOUD_RUNTIME = 'nodejs18.x'
```

> If the `cloudRuntime` key is not defined, then {{ PRODUCT }} will detect your project's Node.js version upon running `{{ CLI_CMD(deploy) }}`. If an unsupported version is detected, then it will set your version to `nodejs18.x`. For instance:

Reviewer: OK. What do you think about explicitly indicating that this behavior is restricted to a specific version? 
```suggestion
If the `cloudRuntime` key is not defined, then {{ PRODUCT }} will detect your project's Node.js version upon running `{{ CLI_CMD(deploy) }}`. If an unsupported version is detected when using {{ PRODUCT }} v7.4.0 or later, then it will set your version to `nodejs18.x`. For instance:
```

```bash

> # Valid Node.js 16 runtime for {{ PRODUCT }} v7.4.0 through v7.4.4

Reviewer: ```suggestion
# The Node.js 16 runtime is valid for {{ PRODUCT }} v7.4.4 or earlier
```

$ node --version
v16.19.1
$ {{ CLI_CMD(deploy) }}

...

> # Unsupported Node.js 21 runtime, defaults to Node.js 18 with a warning

Reviewer: If the default varies by version, then we should mention it here. Something like:
```suggestion
# Unsupported Node.js 21 runtime; defaults to Node.js 18 with a warning for {{ PRODUCT }} v7.4.0 or later
```
